changelog.txt

```diff
@@ -17,6 +17,7 @@ Added
 - Schema module for the ProjectContext (#110).
 - ProjectContext support for the DocumentList, ImageViewer, TextDisplay, and VideoViewer modules (#110).
 - Dashboard config option ``CARDS_PER_ROW`` controls the number of cards per row in the desktop view (#133).
+- The endpoint ``views.get_file`` now reads the ``download_name`` request argument (#127).
 
 Changed
```

Comment:
This is a minor issue, but does `Changed` have to be lined up with Added or Removed?